docs/fastdds/dynamic_types/discovery_matching.rst

```diff
@@ -0,0 +1,66 @@
+.. include:: ../../03-exports/aliases-api.include
+.. include:: ../../03-exports/aliases.include
+
+.. _dynamictypes_discovery_matching:
```

Comment:
This section has nothing to do with "Dynamic Types". Only the discovery of Type information needed for matching. I won't mention "Dynamic Types" here.

For example, this title should be "Discovery and Endpoint Matching".

Reply:
fixed in [e6714a3](https://github.com/eProsima/Fast-DDS-docs/pull/730/commits/e6714a3c5febe9d35a953b5c4f57fb084ed94a81)